app/oxalis/security/WebknossosBearerTokenAuthenticatorService.scala

```diff
@@ -76,7 +77,8 @@ class WebknossosBearerTokenAuthenticatorService(settings: BearerTokenAuthenticat
     for {
       tokenAuthenticator <- repository.findOneByValue(tokenValue) ?~> "auth.invalidToken"
       _ <- bool2Fox(tokenAuthenticator.isValid) ?~> "auth.invalidToken"
-      user <- userService.findOneByEmail(tokenAuthenticator.loginInfo.providerKey)
+      idValidated <- ObjectId.parse(tokenAuthenticator.loginInfo.providerKey) ?~> "auth.tokenNoId" // TODO readable message, warning that token may be old format (or support old format with disambiguation?)
```

Comment:
TODO reminder

Reply:
Fortunately we found a way to migrate the legacy tokens (they now point to the user id instead of the email) so no legacy-specific error message is required :) I’ll just use the normal error message here too.